lib/OpenQA/WebAPI/Controller/API/V1/JobTemplate.pm

```diff
@@ -236,22 +266,29 @@ sub update {
     my $machines              = $schema->resultset('Machines');
     my $test_suites           = $schema->resultset('TestSuites');
     my $products              = $schema->resultset('Products');
-    foreach my $group (keys %{$yaml}) {
+    {
```

Comment:
Keeping the indentation level makes this easier to review. But I'd prefer to get rid of it before the final merge.

Reply:
Yeah, that was my intention, apparently my comment got lost when editing the description. I wish such a hack wasn't needed... I hate destroying diffs with indentation changes...